pypesto/visualize/parameters.py

```diff
@@ -471,6 +430,58 @@ def handle_inputs(
     return lb, ub, x_labels, fvals_out, xs_out
 
 
+def _handle_inner_inputs(
+    result: Result,
+):
```

Comment:
return type would be

```python
(Union[None, list(float])
```
for each of inner_xs, inner_xs_names, inner_lb, inner_ub?

Reply:
I think it's 
```
Union[
    Tuple[None, None, None, None],
    Tuple[list[np.ndarray], list[str], np.ndarray, np.ndarray],
    ]
```
Since it either returns 4 None's or 4 things